packages/react-dom-bindings/src/client/ReactDOMFloatClient.js

```diff
@@ -453,6 +453,11 @@ export function getResource(
     );
   }
   switch (type) {
+    case 'noscript': {
+      // On the client we never need to render noscript because if React is running
+      // we are in an environment that supports scripts
+      return null;
+    }
```

Comment:
This may be too extreme. some people may use it as an inert marker element. Should we render the node but not it's children? this is harder b/c we'd have to add something to a hot path for HostComponent which we don't want to do. I'd rather not use HostResource though if they need to be placed in the document because I don't want to have to identify the right insertion point.

Reply:
this no longer applies, i removed noscript as a resource type since we already handled not rendering it on the client